src/main/java/duke/MainWindow.java

```diff
@@ -0,0 +1,67 @@
+package duke;
```

Comment:
Would it be better organize these classes under the duke package into further subcategories?